.env

```diff
@@ -1,2 +1,3 @@
-FLOW_ORGANIZATION='playground'
-FLOW_API_KEY='HlGgfflLamiTQJ'
+FLOW_ORGANIZATION=playground
+FLOW_API_KEY=HlGgfflLamiTQJ
```

Comment:
yikes!! we should never hardcode API keys here - please delete that API in console and refactor this